assistant_library_with_local_commands_demo.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run a recognizer using the Google Assistant Library.
+
+The Google Assistant Library has direct access to the audio API, so this Python
+code doesn't need to record audio. Hot word detection "OK, Google" is supported.
+
+The Google Assistant Library can be installed with:
+    env/bin/pip install google-assistant-library==0.0.2
+
+It is available for Raspberry Pi 2/3 only; Pi Zero is not supported.
+"""
+
+import logging
+import subprocess
+import sys
+
+import aiy.assistant.auth_helpers
+import aiy.audio
+import aiy.voicehat
+from google.assistant.library import Assistant
+from google.assistant.library.event import EventType
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] %(levelname)s:%(name)s:%(message)s"
+)
+
+
+def power_off_pi():
+	aiy.audio.say('Good bye!')
+	subprocess.call('sudo shutdown now', shell=True)
+
+
+def reboot_pi():
+	aiy.audio.say('See you in a bit!')
+	subprocess.call('sudo reboot', shell=True)
+
+def your_ip_address():
+        shell_command = "ip route get 1 | awk '{print $NF;exit}'"
+        output = subprocess.getoutput(shell_command)
+        aiy.audio.say(output)
```

Comment:
Maybe it is an option to point out that audio.say can take 2 argumants, so using ie  `aiy.audio.say(output,lang="en-US")`